templates/includes/sidebar_login.html

```diff
@@ -0,0 +1,12 @@
+<div class="text-end">
+
+  {% if user.is_authenticated %}
+    <button type="button" class="btn btn-outline-primary">
+      <a href="{% url 'logout' %}" class="custom-btn">Logout</a>
+    </button>
```

Comment:
Avoid nesting `<a>` tags inside `<button>` elements, as this is not valid HTML. Consider using an `<a>` tag styled as a button or handle the click event with JavaScript.